src/Tools/MicroBuild/Build.proj

```diff
@@ -24,7 +24,6 @@
     <MSBuild Projects="$(ProjectDir)src\Dependencies\Dependencies.sln" BuildInParallel="true" />
     <MSBuild Projects="$(ProjectDir)src\Setup\SetupStep1.proj" />
 
-    <!-- TODO: pass arguments -->
     <Exec Command="$(ProjectDir)Binaries\$(Configuration)\SignRoslyn\SignRoslyn.exe $(SignRoslynArgs) -binariesPath &quot;$(BinariesPath)&quot;" WorkingDirectory="$(ProjectDir)" />
 
     <MSBuild Projects="$(ProjectDir)src\NuGet\NuGet.proj" />
```

Comment:
This is the other location that `NuGet.proj` was built from.  You can see one line below where it immediately calls `SetupStep2.proj` which started with another build to `NuGet.proj`.  That is the line deleted above in `SetupStep2.proj`